### PR TITLE
Polish language support

### PR DESCRIPTION
**Pull request by wozniakpl**
Added polish lexer, some polish law implemented in polish.

The text that this code is based on: https://isap.sejm.gov.pl/isap.nsf/download.xsp/WDU20000860959/U/D20000959Lj.pdf

Things left to do:
- Add syntax highlighting for vscode
- Add polish diacritics - some are done but these that were never used yet, are left undone

### CODE REVIEW

File: examples/polish_taxes/podatek_od_czynnosci_cywilnoprawnych/rozdzial_3.catala_pl [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+zakres UmowaSprzedazy:
+  definicja podatek wynosi
+    kwota *$ stawka
+
+zakres UmowaSprzedazy:
+  etykieta a7_u1_p1_ppa
+  definicja stawka wynosi 2%
+```
+> End metadata
```
Comment (denismerigoux) [inline examples/polish_taxes/podatek_od_czynnosci_cywilnoprawnych/rozdzial_3.catala_pl:24]: You can also translate that in Polish in the lexer no ?

File: examples/polish_taxes/polish_taxes.catala_pl [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Master file
```
Comment (denismerigoux) [inline examples/polish_taxes/polish_taxes.catala_pl:1]: This one you can also translate in Polish

File: examples/polish_taxes/tests/test_a7_u1_p1.catala_pl [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+
+deklaracja zakres Test_A7_U1_P1_PPb:
+  kontekst sprzedaz zakres UmowaSprzedazy
+
+zakres Test_A7_U1_P1_PPb:
+  definicja sprzedaz.kwota wynosi $100
+  definicja sprzedaz.inne_prawa_majatkowe wynosi prawda
+
+  asercja sprzedaz.podatek = $1
```
Comment (denismerigoux) [inline examples/polish_taxes/tests/test_a7_u1_p1.catala_pl:22]: By tweaking the regex in the lexer, you can customize the money amount to use the Polish monetary sign and the Polish number writing convention.

**Review by denismerigoux — COMMENTED** [review]
That looks like a very nice start ! For the diacritics support, you have to use the unicode encoding of the characters -- see the French lexer, we have diacritics too.

**Comment by denismerigoux** [comment]
I fixed some syntax highlighting and lexer issues, it seems to work now : [polish_taxes.pdf](https://github.com/CatalaLang/catala/files/6464820/polish_taxes.pdf) :) Don't forget to delete the `_minted...` folder in `examples/polish_taxes` before trying to recompile the LateX, otherwise you'll get stuck on your previous EmptyString error.

**Comment by wozniakpl** [comment]
When I'm copying the current `syntax_highlighting/pl/catala_pl.iro` to `https://eeyo.io/iro/` , it says that is has the `Invalid Configuration`. 
Screenshot
Do you know what to do with that?

**Comment by denismerigoux** [comment]
I fixed the Iro files, you should be able to generate the other syntax highlighting plugins now :)

**Comment by denismerigoux** [comment]
Another comment : be careful what you choose for the Polish keywords. The French and English keywords have been validated by lawyers to check that they could understand them. "Scope" is also a law concept, and for instance we chose "content" instead of "type" because lawyers told us that this word was more understandable. I suggest you check your choice of keywords with a Polish lawyer :)

**Comment by wozniakpl** [comment]
> I fixed the Iro files, you should be able to generate the other syntax highlighting plugins now :)

Thanks for the help! One more thing - when I copy the current catala_pl.iro to eeyo, the generated pygmentz are different than the current ones. Also, when I paste the new ones, they don't work and I get this error (even though I clear minted stuff and rebuilt everything from `git clean -fxd`ed repo)
```
(./_minted-polish_taxes/80C30ABAC40EEC628B3051E9CAEE8D08F7FE8087781FD914E29CF24
71B8351E1.pygtex
! FancyVerb Error:
  Empty verbatim environment
.
\FV@Error ... {FancyVerb Error:
\space \space #1
}
                                                  
l.17 \end{Verbatim}
```

**Comment by denismerigoux** [comment]
Indeed, this comes from a limitation of https://eeyo.io/iro/ which does not work perfectly well. You have to manually edit some things for the generated file to work well. For `pygments`, you can compare the diff between what Iro gives you and what I committed.

**Comment by denismerigoux** [comment]
FYI, the `0.5% * 1000 PLN = 5 PLN` is passing correctly on my side and the CI.

**Comment by denismerigoux** [comment]
@wozniakpl some unit tests for the Polish inheritance taxes are failing, you can trigger them with commands such as :

    SCOPE=Test_A7_U1_P3 make -C examples/polish_taxes tests/test_a7_u1_p3.run

**Review by denismerigoux — APPROVED** [review]

**Pull request merged**